src/transports/IframeExecutionEnvironmentTransport.ts

```diff
@@ -66,9 +66,32 @@ class IframeExecutionEnvironmentTransport extends Transport {
     this.commandStream.on('data', (data: unknown) => {
       this.transportRequestManager.resolveResponse(JSON.stringify(data));
     });
+    this.transportRequestManager.transportEventChannel.addListener(
+      'notification',
+      this.notificationHandler,
+    );
+    this.transportRequestManager.transportEventChannel.addListener(
+      'error',
+      this.errorNotificationHandler.bind(this),
+    );
     return true;
   }
 
+  private errorNotificationHandler(error: any) {
+    this.notificationHandler({
+      id: null,
+      error: {
+        code: error.code,
+        message: error.message,
+        data: error.data,
+      },
+    });
+  }
+
+  private notificationHandler = (event: any) => {
+    window.parent.postMessage(event, '*');
```

Comment:
What does `window.parent` refer to here? I thought this was running _outside_ of the iframe execution environment :thinking:

Reply:
`window.parent` here is inspector.open-rpc.org 

context:

inspector.open-rpc.org => iframe postmessage => this code (which itself makes an iframe and talks over  postmessage/object multiplex).